### PR TITLE
added a build step for Jellyfin Web

### LINKED ISSUE #127: Contribution guide for server needs updated
The contribution guide for server needs to be update to include the new build step for jellyfin-web.

https://jellyfin.readthedocs.io/en/latest/contributor-docs/contributing-code/#set-up-your-copy-of-the-repo

https://github.com/jellyfin/jellyfin-docs/blob/master/general/contributing/development.md

### PR DESCRIPTION
**Pull request by henrick-tissink**
Fixes #127.

### CODE REVIEW

File: general/contributing/development.md [hunk 1]
```diff
@@ -34,47 +34,51 @@ The first step is to set up a copy of the Git repository of the project you want
...
 
 1. Initialize the Git submodules; most projects have at least one:  
-    `git submodule update --init`
+   `git submodule update --init`
 
     **NOTE:** Avoid editing files inside of submodules; this may result in unintended side-effects including overwriting of your work. Always modify the submodule in its own cloned repository elsewhere.
 
+1. To get the `Jellyfin.Server` project to run successfully, checkout both the main [Jellyfin project](https://github.com/jellyfin/jellyfin), as well as the [Jellyfin Web](https://github.com/jellyfin/jellyfin-web) project.
+
+1. Build the Jellyfin Web project with Yarn or NPM, and copy the location of the resulting `dist` folder. In your `Jellyfin.Server` project add an environment variable named `JELLYFIN_WEB_DIR` with the value set to the full path of your `dist` folder.
```
Comment (Bond-009) [inline general/contributing/development.md]: You have to do either one, not both.

Comment (henrick-tissink) [reply general/contributing/development.md]: Do you mind elaborating? I had to follow all of those steps to get the server running successfully from VS.

Comment (Bond-009) [reply general/contributing/development.md]: You can either copy the ‘dist’ folder into the ‘MediaBrowser.WebDashboard’ project (don’t know the exact path rn) or just set the env variable to the path of the ‘dist’ directory.

Comment (henrick-tissink) [reply general/contributing/development.md]: Ah yes, I guess I can clarify it a bit better - I meant "just set the env variable to the path of the ‘dist’ directory."

Comment (Bond-009) [reply general/contributing/development.md]: Oh, I see now, “copy the **location**”

Comment (henrick-tissink) [reply general/contributing/development.md]: Yep

**Pull request merged**